app/controllers/admin/domains_controller.rb

```diff
@@ -65,6 +67,8 @@ def update
     def versions
       @domain = Domain.where(id: params[:domain_id]).includes({ versions: :item }).first
       @versions = @domain.versions
+      @last_version = @versions.last
+      @old_versions = Kaminari.paginate_array(@versions.not_creates.reverse).page(params[:page]).per(DEFAULT_VERSIONS_PER_PAGE)
```

Comment:
Do our user have any ability to input a number of entries per page? If so I'd suggest to use something like 
params[:per_page] || DEFAULT_VERSIONS_PER_PAGE , maybe moved to some helper method.
If not- ok then, we are good to go

Reply:
No, there is no per page input, not necessary